src/lib/driver/message_handler.ts

```diff
@@ -29,16 +29,18 @@ export class DriverMessageHandler {
       case DriverCommand.getLogConfig:
         return { config: dumpLogConfig(driver, client.schemaVersion) };
       case DriverCommand.updateLogConfig:
+        const currentLogConfig = dumpLogConfig(driver, client.schemaVersion);
         driver.updateLogConfig(message.config);
-        clientsController.clients.forEach((cl) => {
-          if (cl !== client) {
+        const newLogConfig = dumpLogConfig(driver, client.schemaVersion);
+        if (currentLogConfig !== newLogConfig) {
```

Comment:
This will always be `true` because `dumpLogConfig` creates a new object. You will need to use a deep equal utility. 

Honestly, I just wouldn't bother and just send it.

Reply:
OK, how about the other point of sending the original client the message?